frontend/src/lib-components/molecules/date-picker/DatePickerLowLevel.tsx

```diff
@@ -269,6 +290,16 @@ export default React.memo(function DatePickerLowLevel({
         minDate={minDate}
         maxDate={maxDate}
       />
+      <StyledIconButton
```

Comment:
I would take a quick look at the employee side too to check if the increased with of this component broke some layouts e.g. in modals where the space is limited

Reply:
Good point - is there a specific modal where it is an issue that should be checked?

Reply:
Not sure 🤔 Creating a new document template (lomakepohjat ylävalikosta) could be one. Maybe just use 10min to randomly poke around. Not that big of a deal anyway.

Reply:
Seems like there's "enoughish"(?) space on the citizen modal and e.g. the employee new doc template ->

<img width="997" alt="Screenshot 2024-12-12 at 15 12 56" src="https://github.com/user-attachments/assets/2aeccc36-4278-4404-a04d-6cd60d35b206" />
<img width="399" alt="Screenshot 2024-12-12 at 15 14 31" src="https://github.com/user-attachments/assets/e4623655-d472-4405-bb71-ce4d7d2100fe" />